YoCode/FeatureDetailsStorage.cs

```diff
@@ -1,172 +1,216 @@
-﻿using System;
+﻿using Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
-using System.Text;
+using System.IO;
+using System.Linq;
 
 namespace YoCode
 {
     class FeatureDetailsStorage
     {
-        public Dictionary<Feature,FeatureDetails> InitializeJuniorDetails()
+        Dictionary<Feature, double> WeightingsFromJson;
+        TestType mode;
+
+        string JuniorPath = @"..\..\..\FeatureWeightings\JuniorWeightings.json";
+        string OriginalPath = @"..\..\..\FeatureWeightings\OriginalWeightings.json";
```

Comment:
One downside of this is that we now have extra files that we need to ship with YoCode, right? We must remember to include them in the zip file. In fact, do these paths need to be in `appsettings.json`? I can imagine they will be different when we package them in the zip file.

Reply:
These are stored in FeatureWeightings folder.
![image](https://user-images.githubusercontent.com/7968437/44533071-9c535600-a6ec-11e8-9626-b34b0529c3e0.png)

It contains two .json files, JuniorWeightings and OriginalWeightings. Why is that a problem?

Reply:
Where will they go in the zip file when we release YoCode? Will `..\..\..\FeatureWeightings` be the right directory in that case?

Reply:
Probably not... What's the possible solution?

Reply:
`appsettings.json`. Get the name of the directory from `appsettings.json`, then the zip file can contain an `appsettings.json` that we've already set up with the right paths. The weightings files can be in git just as you've done, that's fine, but the path that YoCode uses to find them should come from `appsettings.json`.

Reply:
It was easier when nobody had suggested being able to edit the weightings. ;)

Reply:
Nothing that is worth doing is easy 😁